server/sonar-server/src/test/java/org/sonar/server/qualitygate/ws/CreateActionTest.java

```diff
@@ -94,6 +96,29 @@ public void create_quality_gate_with_organization() {
     assertThat(qualityGateDto).isNotNull();
   }
 
+  @Test
+  public void creating_a_qg_with_a_name_used_in_another_organization_should_work() {
```

Comment:
This test is not clear => 2 execution of the web service, implying to have permission on 2 organizations.

I would expect to : 
1. Create an organization1, and create a quality gate inside
2. Create an organization2
3. Execute the request to create a quality gate in organization2, using name of quality gate created in organization1

Reply:
done